tests/test_dask_image/test_ndinterp/test_affine_transformation.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pytest
+
+import dask_image.ndinterp as da_ndinterp
+
+import numpy as np
+import dask.array as da
+
+
```

Comment:
Are you able to add a couple more test cases to cover the currently untested lines of your function?
https://coveralls.io/builds/34077648/source?filename=dask_image/ndinterp/__init__.py

Things like:
* no output shape specified
* no order specified
* if order is not an integer value
* Do we get the user warnings we expect for prefiltering, etc. if we pass in those keyword arguments, using [pytest warns](https://docs.pytest.org/en/3.0.3/recwarn.html)

Reply:
Good point! Actually, `output_shape=None` had lead to an exception. That's fixed and more tests have been added. Let me know if you think some more would make sense.